docker-compose.yml

```diff
@@ -48,12 +49,46 @@ services:
       start_period: 40s
   
   logspout:
+    container_name: logspout
     restart: on-failure
     image: gliderlabs/logspout
     volumes:
       - "/var/run/docker.sock:/var/run/docker.sock"
-    container_name: logspout
     command: syslog+tls://${ELK_ENDPOINT}
+  
+  cadvisor:
+    container_name: siyuan-cadvisor
+    restart: on-failure
+    networks: 
+      - monitor-net
+    image: gcr-io.mirrors.sjtug.sjtu.edu.cn
```

Comment:
You will have to specify the image to pull. Just replace gcr.io with this domain, and append the image name.